src/python_testing/TC_CADMIN_1_22_24.py

```diff
@@ -104,13 +106,14 @@ async def test_TC_CADMIN_1_22(self):
         except ChipStackError as e:
             # Since we provided 901 seconds as the timeout duration,
             # we should not be able to open comm window as duration is too long.
+            # we are expected receive Failed to open commissioning window: IM Error 0x00000585: General error: 0x85 (INVALID_COMMAND)
             asserts.assert_equal(e.err,  0x00000585,
```

Comment:
```suggestion
            _INVALID_COMMAND = 0x00000585
            asserts.assert_equal(e.err,  _INVALID_COMMAND,
```

Then you can replace all the other values of `0x00000585` with `_INVALID_COMMAND` and it makes the code more readable

Reply:
Thank you this does make the code much easier to read, i will update using your suggestion above!